frontend_app_logbook/src/component/Logbook.jsx

```diff
@@ -179,6 +182,18 @@ export class Logbook extends React.Component {
     })
   }
 
+  handleExpand = () => {
+    this.setState({
+      expandAll: true
+    })
+  }
+
+  handleCollapse = () => {
+    this.setState({
+      expandAll: false
+    })
+  }
```

Comment:
try handling the state of expanded or not in a specific property on each logbook entries

Reply:
done